app/uploaders/avatar_uploader.rb

```diff
@@ -0,0 +1,13 @@
+class AvatarUploader < CarrierWave::Uploader::Base
+  include CarrierWave::MiniMagick
+
+  storage :file
```

Comment:
It's bad idea to store files on running server. Try to use some remote cloud storage (e.g. aws s3).

Reply:
can I use another droplet on digital ocean for store files?

Reply:
I don't think that it's a good idea. In real world you will use cloud storage for files. You need to  get used to it now. Is it a problem for you to register trial s3 account that is free for 1 year of usage?